instrumentation/basic_meter.go

```diff
@@ -0,0 +1,39 @@
+package instrumentation
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+type basicMeter struct {
+	name   string
+	start  int64
+	end    int64
+	logger BasicLogger
+
+	params []*Field
+}
+
+type BasicMeter interface {
+	Done()
+}
+
+func (m *basicMeter) Done() {
+	m.end = time.Now().UnixNano()
+	diff := float64(m.end-m.start) / NanosecondsInASecond
+
```

Comment:
I don't understand where start is initialized?

Reply:
When we call `logger.Meter()`